util/Migrator/DbScripts/2021-04-07_00_IncreaseOrgSeatSize.sql

```diff
@@ -0,0 +1,226 @@
+ALTER TABLE [dbo].[Organization]
```

Comment:
Please add a check before altering this column to determine if the type is explicitly `SMALLINT` before changing it (say a year from now we changed it to `BIGINT`, we wouldn't want to regress, etc.)

e.g.
```sql
IF EXISTS (SELECT * FROM INFORMATION_SCHEMA.COLUMNS WHERE COLUMN_NAME = 'Seats' AND DATA_TYPE = 'smallint' AND TABLE_NAME = 'Organization')
BEGIN
    ....
END
```